pkg/virtctl/guestfs/guestfs.go

```diff
@@ -548,6 +548,13 @@ func createLibguestfsPod(pvc, image, cmd string, args []string, kvm, isBlock boo
 					Stdin:           true,
 					TTY:             true,
 					Resources:       resources,
+					ReadinessProbe: &corev1.Probe{
+						ProbeHandler: corev1.ProbeHandler{
+							Exec: &corev1.ExecAction{
+								Command: []string{"/bin/cat", appliancePath + "/done"},
```

Comment:
@maya-r could you use the env variable LIBGUESTFS_PATH instead of `appliancePath`, like `${LIBGUESTFS_PATH}/done`

Reply:
mh if it is an exec, it should work as basically the functional tests are doing the same. I'll give a try later